core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -63,10 +68,10 @@ object MultiCloudWorkspaceService {
   * This service knows how to provision a new "multi-cloud" workspace, a workspace managed by terra-workspace-manager.
   */
 class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
-                                 workspaceManagerDAO: WorkspaceManagerDAO,
+                                 val workspaceManagerDAO: WorkspaceManagerDAO,
```

Comment:
What does it mean if there is no val? Should `billingProfileManagerDAO` also be fixed?

Reply:
I don't really know tbh. I think a `val` on a property means that it's public. I'm using this in testing to mock some return values from the workspace manager dao